#
#

Header header
bool emergencyStatus   # true : set; false : clear;
